### PR TITLE
Add paragraph about the Prisma Data Proxy to connection management

### PR DESCRIPTION
**Pull request by 2color**

### CODE REVIEW

File: content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx [hunk 1]
```diff
@@ -298,16 +298,18 @@ datasource db {
...
+
+### The Prisma Data Proxy
+
+The [Prisma Data Proxy](/concepts/components/prisma-data-platform#prisma-data-proxy) is a managed external connection pooler built by Prisma that is integrated in the [Prisma Data Platform](https://www.prisma.io/dataplatform) and handles connection pooling for you.
 
 ### PgBouncer
 
 PostgreSQL only supports a certain amount of concurrent connections, and this limit can be reached quite fast when the service usage goes up – especially in [serverless environments](#serverless-environments-faas).
 
 PgBouncer holds a connection pool to the database and proxies incoming client connections by sitting between Prisma Client and the database. This reduces the number of processes a database has to handle at any given time. PgBouncer passes on a limited number of connections to the database and queues additional connections for delivery when connections becomes available.
```
Comment (janpio) [inline content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx]:
```suggestion
[PgBouncer](https://www.pgbouncer.org/) holds a connection pool to the database and proxies incoming client connections by sitting between Prisma Client and the database. This reduces the number of processes a database has to handle at any given time. PgBouncer passes on a limited number of connections to the database and queues additional connections for delivery when connections becomes available.
```

File: content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx [hunk 2]
```diff
@@ -298,16 +298,18 @@ datasource db {
...
 
-To use PgBouncer, see: [Configure Prisma Client with PgBouncer](configure-pg-bouncer)
```
Comment (janpio) [inline content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx:309]: Why remove this link? I think it should stay as that configuration is not obvious. (Maybe removed accidentally)

Comment (tanberry) [reply content/300-guides/100-performance-and-optimization/150-connection-management/index.mdx:309]: @2color let's leave that link but if you have a reason for removing it, I'll do that.

**Comment by tanberry** [comment]
Thank you @2color !!

**Pull request merged**